### ISSUE TITLE
wrong typescripts files

### ISSUE THREAD

**Issue by JLarky** [sergeyt/karma-typescript-preprocessor#25]
Something really strange is going on
```
./node_modules/karma/bin/karma start tests/unit/karma.conf.js 
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
error TS6053: File 'start.ts' not found.
error TS6054: File 'tests/unit/karma.conf.js' must have extension '.ts' or '.d.ts'.
```

```
$ xvfb-run gulp karma                                                                                                                                                                                 
[18:34:53] Using gulpfile /x/w/e/l/src/web/gulpfile.js
[18:34:53] Starting 'karma'...
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
error TS6053: File 'karma.ts' not found.
```

```
$ xvfb-run gulp karma_watch
[18:35:17] Using gulpfile /x/w/e/l/src/web/gulpfile.js
[18:35:17] Starting 'karma_watch'...
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
error TS6053: File 'karma_watch.ts' not found.
```

so whatever I pass as a parameter to gulp or karma gets sent to typescript command line?

unfortunately I don't have exact versions of dependencies, but I first downloaded karma about three weeks ago and updated (rm -r node_modules) today.

**Comment by sergeyt** [comment]
@JLarky please let me know what version of typescript do you use (you should add it dependencies). Last time I checked the preprocessor with tsc v1.3. The main problem with typescript that it has no nodejs API (like coffee-script does), due that reason [node-tsc](https://github.com/sergeyt/node-tsc) was created which attempts to patch tsc.js code to avoid auto run and even creates [vm](http://nodejs.org/api/vm.html) sandbox to expose as nodejs module. Such kind of hacks could not be stable, i.e. that means for example node-tsc could not work for typescript v1.4. You could try to use typescript v1.3 to check whether the preprocessor works with it and detect issue when use typescript v1.4.

**Comment by JLarky** [comment]
yes, you are right, downgrading typescript from 1.4 to 1.3 helps!

**Comment by sergeyt** [comment]
@JLarky I've fixed support for typescript v1.4 in [node-tsc](https://github.com/sergeyt/node-tsc/commit/be44e17dcfe2ed42ee69b4e2d7f8ac4811d85248). Try to use karma-typescript-preprocessor v0.0.15.
